src/ci/bin/test.inc.sh

```diff
@@ -475,7 +482,7 @@ cromwell::private::create_centaur_variables() {
             CROMWELL_BUILD_CENTAUR_TEST_DIRECTORY="${CROMWELL_BUILD_CENTAUR_RESOURCES}/${CROMWELL_BUILD_CENTAUR_TYPE}TestCases"
             if test "${CROMWELL_BUILD_BACKEND_TYPE}" = "papi_v2alpha1" || test "${CROMWELL_BUILD_BACKEND_TYPE}" = "papi_v2beta"
             then
-              CROMWELL_BUILD_CENTAUR_CONFIG="${CROMWELL_BUILD_RESOURCES_DIRECTORY}/papi_v2_centaur_application.conf"
```

Comment:
Is papi_v2_centaur_application.conf still is in use somewhere? If not, probably it should it be deleted.

Reply:
this was just a renaming (see below `...esources/papi_v2_centaur_application.conf → ...esources/centaur_application_papi_v2.conf`)